src/modules/ie-objects/ObjectDetailPage.tsx

```diff
@@ -1487,11 +1490,10 @@ export const ObjectDetailPage: FC<DefaultSeoInfo> = ({ title, description, image
 									setIsOcrEnabled(!isOcrEnabled);
 								}}
 							>
-								<Highlighter
-									searchWords={searchOcrText.split(' ')}
-									autoEscape={true}
-									textToHighlight={textLocation.text}
-								/>
+								{highlighted(
```

Comment:
This should not be a function, it always outputs the same component.

Reply:
nice!
finally a "split this into a component" remark that we can agree on 😄 

i've moved it to a separate component <HighlightSearchTerms toHighlight={text} />